README.md

```diff
@@ -1,42 +1,53 @@
-### FishBrain Catches Master/Detail
+### Fishbrain Assignment
 
 The goal of this project is to assess Android development skills. It is designed to be challenging, and evaluate all aspects of development.
 
-- **Project Name:** FIBCatches
-- **Project Goal:** Creating a simplified version of the FishBrain app that just includes a simplified catches master/detail view.
+- **Project Name:** Fishbrain Assignment
+- **Project Goal:** Creating a simplified version of the Fishbrain app that just includes a simplified catches view.
 - **Technology:** Java
-- **Deliverables:** The solution should be sent using the greenhouse.io link that will be provided during the interview process.
+- **Deliverables:** The solution should be sent as a pull request in this github project
```

Comment:
I was thinking to have that private, or make PR private - I remember I saw that we can do that. Let me check again